sdk/storage/storage-internal-avro/rollup.config.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Do we need this file? Seems you only need rollup.test.config.js for testing purpose

Reply:
We don't need it for now.
Do you think we should the script for build in the package.json as well? I thought we may want to extract the API.